data/Tsakalos_2022/metadata.yml

```diff
@@ -0,0 +1,328 @@
+    nominal CGO trait categories (i.e. lignotubers, herbaceous and woody rhizomes,
+    stolons, tubers, stem fragments) and recognizing the absence of clonality (i.e.
+    nonclonal) as an additional category, for all 463 species present within the 138
+    plots (Supplementary Data Table S1). Clonality is the potential capacity to produce
+    multiple rooting units (ramets), which may become independent when the connection
+    with the parent plant is lost. We combined new field observations (i.e. ~25 %
+    of the 463 species) made during the peak (spring) flowering season of 2019 and
+    2020 with our DRYAD database (Tsakalos et al., 2020) and information from the
+    FloraBase database (Western Australian Herbarium, 1998-) to establish the presence
+    or absence of clonal traits. When we used the FloraBase database, we assigned
+    trait values using information from specimens near the study area. With a few
+    exceptions, the applied classification system of CGOs followed the concepts and
+    nomenclature of standard protocols (Klimešová et al., 2019). Our rhizomatous traits
+    differed in two ways. First, we did not distinguish where their growth originated
+    (i.e. aboveor belowground) because we did not have such information. Second, we
+    have distinguished CGO classes for herbaceous and woody rhizomes because of the
+    differences in their longevity and ability to produce vertical stems (Pausas et
+    al., 2018). Our tuber CGO is a large category and encompasses Klimešová et al.'s
+    (2019) tubers and bulbs and includes a combination of (herbaceous) stem tubers,
+    root tubers and bulbs. Lastly, we have classified lignotubers (including xylopodium;
+    see Pausas et al., 2018) as a CGO because lignotubers have the potential to become
+    independent rooting units (ramets), as observed in several Banksia species (Proteaceae;
+    see also Millar et al., 2010) in our study area. A selection of CGOs is presented
+    in Fig. 1, and a description of their general morphological characteristics and
+    ecological relevance is given in Table 1.
+substitutions:
+- trait_name: bud_bank_location
+  find: HR
+  replace: rhizome_non_woody
+- trait_name: bud_bank_location
+  find: WR
+  replace: rhizome_woody
+- trait_name: bud_bank_location
+  find: S
+  replace: stolon
+- trait_name: bud_bank_location
+  find: L
+  replace: basal_stem_buds
+- trait_name: bud_bank_location
+  find: T
+  replace: stem_tuber
+- trait_name: bud_bank_location
+  find: F
+  replace: .na
+- trait_name: bud_bank_location
+  find: NC
+  replace: .na
+- trait_name: clonal_spread_mechanism
+  find: HR
+  replace: rhizome
+- trait_name: clonal_spread_mechanism
+  find: WR
+  replace: rhizome
+- trait_name: clonal_spread_mechanism
+  find: S
+  replace: stolon
+- trait_name: clonal_spread_mechanism
+  find: L
+  replace: .na
+- trait_name: clonal_spread_mechanism
+  find: T
+  replace: stem_tuber
```

Comment:
Both `root_tuber` and `stem_tuber` according to my other comment?

Reply:
I've replace with the more generic `belowground_clonal`